build_tools/python/benchmark_suites/iree/x86_64_benchmarks.py

```diff
@@ -84,14 +87,25 @@ def generate(
         host_environment=common_definitions.HostEnvironment.LINUX_X86_64)
 
```

Comment:
Maybe as CUDA benchmarks, add a comment to explain why the tag `x86_64` needs to present.

Reply:
Added